### PR TITLE
Consolidates cyborg modules into one

### PR DESCRIPTION
**Pull request by tf-4**
This will need some more testing to be sure I didn't break ~~much~~ anything

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -5355,9 +5351,13 @@
 #include "modular_skyrat\modules\customization\modules\surgery\organs\tails.dm"
 #include "modular_skyrat\modules\customization\modules\surgery\organs\vox.dm"
 #include "modular_skyrat\modules\customization\modules\surgery\organs\wings.dm"
-#include "modular_skyrat\modules\cyborg\code\borgpowertools.dm"
 #include "modular_skyrat\modules\cyborg\code\mechafabricator_designs.dm"
+#include "modular_skyrat\modules\cyborg\code\robot_items.dm"
```
Comment (tf-4) [inline tgstation.dme]:
```suggestion
```
this isn't cursed

**Comment by MitztheKat** [comment]
hang on, we adding new cool 2-in-1 tools, or were they already in?

**Comment by tf-4** [comment]
This has no new content

**Comment by tf-4** [comment]
cba to finish this

**Pull request closed without merging**